Representable/representable
# add tags

**changes**
- adds ability for users to tag their community out of a base set of tags
- can either click on a button of the top 15 most commonly used tags, or can type your own (must be in initial set of tags, though)
- displays tags as badges on submission and map pages

**to test**
- python manage.py migrate
- execute new command "addtags" by typing into console: python manage.py addtags
- check that you see success message in console
- create a community with tags
- submit community - check tags appear as expected on submission and map pages

**screenshots**
![image](https://user-images.githubusercontent.com/41943646/125131863-1593fb00-e0c9-11eb-8236-20f5994061b8.png)
![image](https://user-images.githubusercontent.com/41943646/125131945-3bb99b00-e0c9-11eb-9aee-151e44b7642a.png)
![image](https://user-images.githubusercontent.com/41943646/125131981-4411d600-e0c9-11eb-9234-c4cb2f568b1e.png)
